internal/network/command_transit_gateway_attachment.go

```diff
@@ -0,0 +1,130 @@
+package network
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type transitGatewayAttachmentCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type transitGatewayAttachmentHumanOut struct {
+	Id                         string `human:"ID"`
+	Name                       string `human:"Name"`
+	NetworkId                  string `human:"Network ID"`
+	RamShareArn                string `human:"RAM Share ARN"`
+	TransitGatewayId           string `human:"Transit Gateway ID"`
+	Routes                     string `human:"Routes"`
+	TransitGatewayAttachmentId string `human:"Transit Gateway Attachment ID,omitempty"`
```

Comment:
which attribute does this one refer to? this field isn't available in API spec

![Screen Shot 2023-09-08 at 9 50 05 AM](https://github.com/confluentinc/cli/assets/108823534/439f3f0b-4e2f-4075-9a93-e8df879b8e3e)

Reply:
It refers to `status.cloud.transit_gateway_attachment_id`
![image](https://github.com/confluentinc/cli/assets/135164550/ee56e52b-a38d-4bf1-a970-0286440aa0b7)

Reply:
ah got it. this refers to the aws resource.
Should we prefix `Aws` to it - like `AwsTransitGatewayAttachmentId`